python/hail/utils/hadoop_utils.py

```diff
@@ -36,7 +36,9 @@ def hadoop_read(path, buffer_size=8192):
     :return: Iterable file reader.
     :rtype: `io.BufferedReader <https://docs.python.org/2/library/io.html#io.BufferedReader>`_
     """
-    return io.BufferedReader(HadoopReader(path), buffer_size=buffer_size)
+    # return HadoopBinaryDecoder(io.BufferedReader(HadoopReader(path), buffer_size=buffer_size))
+    # return io.BufferedReader(HadoopReader(path), buffer_size=buffer_size)
+    return io.TextIOWrapper(io.BufferedReader(HadoopReader(path, buffer_size), buffer_size=buffer_size), encoding='iso-8859-1')
```

Comment:
Hmm.  I know the old code did this, but it seems sporked to hardcode the encoding.  It takes a default argument encoding=None, I think that's better (it will presumably use the system default determined in some appropriate way).